backend/src/main/java/turip/content/service/ContentService.java

```diff
@@ -1,23 +1,85 @@
 package turip.content.service;
 
+import java.util.List;
+import java.util.stream.Collectors;
 import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 import turip.content.controller.dto.response.ContentCountResponse;
+import turip.content.controller.dto.response.ContentDetailsByRegionResponse;
+import turip.content.controller.dto.response.ContentListByRegionResponse;
+import turip.content.controller.dto.response.ContentWithoutRegionResponse;
+import turip.content.controller.dto.response.TripDurationResponse;
 import turip.content.domain.Content;
 import turip.content.repository.ContentRepository;
 import turip.exception.NotFoundException;
+import turip.tripcourse.service.TripCourseService;
 
 @Service
 @RequiredArgsConstructor
 public class ContentService {
 
     private final ContentRepository contentRepository;
+    private final TripCourseService tripCourseService;
 
     public ContentCountResponse countByRegionName(String regionName) {
         int count = contentRepository.countByRegion_Name(regionName);
         return ContentCountResponse.from(count);
     }
 
+    public ContentListByRegionResponse findContentListByRegionName(
+            String regionName,
+            int size,
+            long lastId
+    ) {
+        // 원하는 size 보다 1개 더 가져와서 더 가져올 수 있는지 확인한다.
+        List<Content> contents = findContentsByRegion(regionName, lastId, size + 1);
+
+        // 실제 반환할 content는 size 만큼 잘라서 반환한다.
+        List<Content> pagedContents = contents.stream()
+                .limit(size)
+                .collect(Collectors.toList());
```

Comment:
a: 자바 16부터는 이렇게도 사용할 수 있어요! 

```suggestion
        List<Content> pagedContents = contents.stream()
                .limit(size)
                .toList();
```

Reply:
해당 부분 반영 완료했습니다!